@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -129,10 +130,11 @@
   border: none;
   padding: 0;
   margin: 0;
-  margin-left: var(--a-spacing-1);
+  margin-left: var(--ax-spacing-1);
   min-width: 10ch;
   width: 100%;
-  height: var(--__ac-combobox-input-height);
+  height: var(--__axc-combobox-input-height);
+  background: transparent;
```

Comment:
```suggestion
  background: transparent;
```
This breaks stuff
<img width="424" alt="Screenshot 2024-11-06 at 08 53 04" src="https://github.com/user-attachments/assets/e48e9e1e-491c-4a6f-b58b-350abde01b80">